extras/test/ConditionTests/on_less_tests.cpp

```diff
@@ -122,7 +122,30 @@ TEST(ConditionTests, handleActionTestsForDouble) {
   // nothing should happen
   channel->setNewValue(25.0);
   cond->handleAction(Supla::ON_CHANGE, action1);
+  
+  // DIMMER use int type on channel value
+  channel->setType(SUPLA_CHANNELTYPE_DIMMER);
 
+  channel->setNewValue(0.0);
```

Comment:
please use setter which is used to set dimmer related values, i.e.:
channel->setNewValue(-1, -1, -1, -1, 0);

Apply this comment to all newly added setNewValue calls

Reply:
done